### PR TITLE
Update .Net tutorial for latest version of Confluent.Kafka

### PR DESCRIPTION
**Pull request by spoplavskiy**
This update switch Confluent.Kafka from v. 0.11.5 to v. 1.1.0. As in 1.1.0 object model has been changed, Worker.cs has been rewritten.
This Confluent.Kafka v. 1.1.0 supports .NetStabdard 1.3, this code can be recompiled for .Net Classic and other, that supports .NetStabdard 1.3

### CODE REVIEW

**Comment by arerlend** [comment]
@spoplavskiy thanks for the contribution!  I'll test it out today and merge it in.

File: quickstart/dotnet/EventHubsForKafkaSample/Worker.cs [hunk 1]
```diff
@@ -49,40 +47,45 @@ public static async Task Producer(string brokerList, string connStr, string topi
...
-                consumer.OnMessage += (_, msg)
-                  => Console.WriteLine($"Received: '{msg.Value}'");
+                BootstrapServers = brokerList,
+                SecurityProtocol = SecurityProtocol.SaslSsl,
+                SaslMechanism = SaslMechanism.Plain,
+                SaslUsername = "$ConnectionString",
+                SaslPassword = connStr,
+                SslCaLocation = cacertlocation,
+                GroupId = consumergroup,
+                BrokerVersionFallback = "1.0.0",        //Event Hubs for Kafka Ecosystems supports Kafka v1.0+, a fallback to an older API will fail
```
Comment (arerlend) [inline quickstart/dotnet/EventHubsForKafkaSample/Worker.cs]: Can we add auto offset reset as earliest?  In the demo scenario, consumer starts after the producer which means a new consumer group's starting offset will be after the end of the demo message sequence.  I know it wasn't there earlier :)

**Comment by arerlend** [comment]
Looks good, thanks again.

File: quickstart/dotnet/EventHubsForKafkaSample/Worker.cs (hunk 1 above)
Comment (spoplavskiy) [reply quickstart/dotnet/EventHubsForKafkaSample/Worker.cs]: Added in commit https://github.com/Azure/azure-event-hubs-for-kafka/pull/55/commits/10fb2a879c72cc7173607e2b373c4dc866508a68

**Pull request merged**